### DISCUSSION TITLE
Please clarify the docs on what should happen when I disable these plugins

### DISCUSSION THREAD

**Discussion by mehov** [gatsbyjs/gatsby#34491] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Summary

From reading the docs, it's not immediately clear to me what will happen if I disable these plugins:

- https://www.gatsbyjs.com/plugins/gatsby-transformer-sharp/
- https://www.gatsbyjs.com/plugins/gatsby-plugin-sharp/
- https://www.gatsbyjs.com/plugins/gatsby-plugin-react-helmet/

Will my Gatsby-powered site stop displaying images? Or will it stop processing the images and instead hotlink them from Wordpress directly? Will it stop showing the meta tags in the header? Or is the helmet plugin only for the custom tags I may want to specify in `gatsby-config`?

All of these are suggested by the sample `gatsby-config.js`. I would like to make my builds as lightweight as possible while you [solve the Parallel Query Memory issues](https://github.com/gatsbyjs/gatsby/discussions/34215), so if I don't need them, I'd rather remove them. But it's not clear to me what would be the downside of that decision.

Please add clarifications to the documentation

**Comment by LekoArts** [comment]
Hi!

Your WordPress plugin has its own setting for downloading & processing images: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/docs/plugin-options.md#htmlusegatsbyimage - if you want to use `gatsby-plugin-image` you'll need to install the necessary peerDependencies. If you just remove the plugins your images will probably stop showing if you don't change anything.
The react-helmet plugin is necessary for putting tags into the HTML ("With this plugin, attributes you add in their component, e.g. title, meta attributes, etc. will get added to the static HTML pages Gatsby builds.").

Removing the plugins won't make your site more lightweight or do anything about memory related issues, as the image processing is running in other processes.
